zerver/tests/test_users.py

```diff
@@ -348,16 +348,6 @@ def test_admin_api_hide_emails(self) -> None:
         self.assertEqual(hamlet["avatar_url"], get_gravatar_url(user.delivery_email, 1))
         self.assertEqual(hamlet["delivery_email"], user.delivery_email)
 
-        # Also verify the /events code path.  This is a bit hacky, but
-        # basically we want to verify client_gravatar is being
-        # overridden.
-        with mock.patch(
-            "zerver.lib.events.request_event_queue", return_value=None
-        ) as mock_request_event_queue:
-            with self.assertRaises(JsonableError):
-                do_events_register(user, get_client("website"), client_gravatar=True)
-            self.assertEqual(mock_request_event_queue.call_args_list[0][0][3], False)
-
```

Comment:
Does whatever this was testing get tested somewhere else now?

Reply:
This was testing whether the `client_gravatar` value passed to `request_event_queue` is overridden. But now we do not change the `client_gravatar` value in `do_events_register` now and we instead check and set it when we actually want to decide whether to include user's avatar url or not.

Reply:
OK, makes sense.